terminator/render/api/api.go

```diff
@@ -21,7 +21,7 @@ type RenderReply struct {
 
 type Renderer func(req RenderRequest) (RenderReply, error)
 
-func HttpRenderHandler(renderer Renderer) func(req *http.Request, w http.ResponseWriter) {
+func HttpRenderHandler(renderer Renderer) func(w http.ResponseWriter, req *http.Request, ) {
 	return func(req http.Request, w http.ResponseWriter) {
```

Comment:
compilation should fail unless you change this, too...maybe we should add CI :P

Reply:
haha yes ... should be fixed now